.gitignore

```diff
@@ -1,3 +1,10 @@
+# fuzzer binaries (we want these at top, because later on we re-ignore things like *.o, .deps, etc)
+# first ignore all
+src/test/fuzz/*
```

Comment:
Began addressing the same issue with #17452, which I needed to add the unignore line to.